doc/connectivity/connectome_extraction.rst

```diff
@@ -105,8 +104,8 @@ of the estimator::
 .. topic:: **Exercise: computing sparse inverse covariance**
    :class: green
 
-   Compute and visualize a connectome on the first subject of the brain
-   development dataset downloaded with :func:`nilearn.datasets.fetch_development_fmri`
+   Compute and visualize a connectome on the first subject of the ADHD
+   dataset downloaded with :func:`nilearn.datasets.fetch_adhd`
```

Comment:
Hum, you have a broken merge :(.

Reply:
Ok, will fix it. Just out of interest, how come it doesn't show up as a merge conflict?